src/fairseq2/models/wav2vec2/setup.py

```diff
@@ -57,7 +57,7 @@ def convert_wav2vec2_checkpoint(
         r"^encoder\.layers\.([0-9]+)\.fc2\.":                 r"encoder.layers.\1.ffn.output_proj.",
         r"^encoder\.layers\.([0-9]+)\.final_layer_norm\.":    r"encoder.layers.\1.ffn_layer_norm.",
         r"^encoder\.embed_tokens\.":                          r"encoder_frontend.embed.",
-        r"^encoder\.pos_conv\.0\.":                           r"encoder_frontend.pos_encoder.conv.",
+        r"^encoder\.pos_conv\.([0-4])\.":                     r"encoder_frontend.pos_encoder.conv.",
```

Comment:
is it possible that there will be multiple pos_conv layers in the same checkpoint ? (pos_conv.0, .. pos_conv.3)

Reply:
Yes, the number of pos_conv layers is determined by the passed in `pos_conv_depth` value, which is 1 by default: https://github.com/fairinternal/fairseq-py/blob/seamless_main/fairseq/models/wav2vec/wav2vec2.py#L258

In dinosr we use 5.

Reply:
I updated this line to 
> r"^encoder\.pos_conv\.([0-4])\.0.":                    r"encoder_frontend.pos_encoder.layers.\1.conv.",

to make ckpt loaded for me.